tests/network-bridge-firewall

```diff
@@ -116,6 +116,35 @@ firewallTests() {
     ! lxc exec c1 -- ping -c1 192.0.2.1 || false
     ! lxc exec c1 -- ping -c1 2001:db8::1 || false
 
+    lxc config device unset c1 eth0 security.mac_filtering
+    lxc stop -f c1
+
+    echo "=> Performing NAT tests with masquerading"
+    lxc start c1
+    sleep 10
+
+    lxc network set lxdbr0 ipv4.nat true
+    lxc network set lxdbr0 ipv6.nat true
+    lxc network create lxdbr1 \
+        ipv4.address=192.0.3.1/24 \
+        ipv6.address=2001:db9::1/64 \
+
+    lxc exec c1 -- ip a add 192.0.2.254/24 dev eth0
+    lxc exec c1 -- ip r replace default via 192.0.2.1 dev eth0
+    lxc exec c1 -- ip a add 2001:db8::254/64 dev eth0 nodad
+    lxc exec c1 -- ip -6 r replace default via 2001:db8::1 dev eth0
+
+    lxc exec c1 -- ping -c1 192.0.3.1
+    lxc exec c1 -- ping -c1 2001:db9::1
+
+    echo "=> Performing NAT tests with explicit SNAT address"
+    lxc network set lxdbr0 ipv4.nat.address 192.0.2.10
+    lxc network set lxdbr0 ipv6.nat.address 2001:db8::10
+
+    lxc exec c1 -- ping -c1 192.0.3.1
+    lxc exec c1 -- ping -c1 2001:db9::1
```

Comment:
Same suggestion about `tcpdump` here. `tests/network-ovn` have such capture examples if needed.

Reply:
Thanks for the pointer I'll have a look. 

Also I am trying to have some multicast tests to properly validate the change in LXD.

Reply:
Fixed.